client/src/styles/trumps/icon.fonts.css

```diff
@@ -438,6 +442,10 @@ Generated using nucleoapp.com
   content: "\ea59";
 }
 
+.icon-utilities::before {
+  content: "\ea06";
+}
+
```

Comment:
However this is solved, do it in `/client/src/styles/trumps/icon.css` instead.

Do not dev in `icon.fonts.css`, because Cortal icon update will replace this file.

Reply:
Example: [the replacement for Cortal v1.4](https://github.com/TACC/Core-Portal/pull/632/files#diff-192cea196a0afaa63e8d904fe8299b5547bbbc0573b0668c095359be331bb715)

Reply:
So I was able to add .icon-utilities::before rule to the icon.css file without issue--however when adding .icon-applications::before rule to the icon.css file, it causes all the category icons to change to the applications icon bc of specificity rules. Trying to think of a workaround